packages/zipkin-instrumentation-axiosjs/package.json

```diff
@@ -6,14 +6,13 @@
   "scripts": {
     "build": "babel src -d lib",
     "test": "mocha --exit --require ../../test/helper.js",
+    "test-browser": "karma start --single-run --browsers ChromeHeadless ../../karma.conf.js",
```

Comment:
cc @ewhauser now this module can pass tests in the browser, even integration tests